modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/VolatilePageMemoryDataRegion.java

```diff
@@ -72,7 +76,12 @@ public class VolatilePageMemoryDataRegion implements DataRegion<VolatilePageMemo
      * Starts the in-memory data region.
      */
     public void start() {
-        VolatilePageMemory pageMemory = new VolatilePageMemory(cfg, ioRegistry, pageSize);
+        int lockConcLvl = IgniteSystemProperties.getInteger(
+                IGNITE_OFFHEAP_LOCK_CONCURRENCY_LEVEL,
+                Integer.highestOneBit(Runtime.getRuntime().availableProcessors() * 4)
```

Comment:
Why is the default value so high, please write a comment about it.****

Reply:
I just moved the code from a different place. I don't know why it's so high, and I don't think that explaining it here would be appropriate

Reply:
Why not use this new constant? 
`OffheapReadWriteLock.DEFAULT_CONCURRENCY_LEVEL`

Reply:
Because it has a different value, obviously. Description here says `Remove PageLockListener from non-test classes`, why would I change settings? Refactoring should not affect the behavior

Reply:
Ok